.gitignore

```diff
@@ -19,3 +19,4 @@ appengine/build
 tools/shows-gen/yt-api-key.json
 src/jekyll/_includes/svgs/
 .bundle/
+google_appengine/
```

Comment:
Yes, the README.md doc instructs contributors to extract the AppEngine SDK into the project root directory. Therefore, I think that we should avoid that the SDK directory is added to git repository with the .gitignore file.